tools/ci/run-examples.sh

```diff
@@ -22,13 +22,15 @@ CURRENT_DIR=$(dirname "${BASH_SOURCE-$0}")
 PROJECT_DIR=$(cd "${CURRENT_DIR}/../.."; pwd)
 
 FLINK_VERSION=1.15.2
+SPARK_VERSION=3.3.1
 
 cd "${PROJECT_DIR}"
 
 wheel_files=`ls ./wheels/*`
 wheel_file=${wheel_files[0]}
 python -m pip install "${wheel_file}"
 python -m pip install "${wheel_file}[flink]"
+python -m pip install "${wheel_file}[spark]"
```

Comment:
This command would override some python libraries used by Flink examples, and leave potential risks. From this aspect, it might be better to install spark dependencies after Flink examples have been completed.

Reply:
Could you please manually trigger the `Nightly Build` workflow like the one below[1], testing that the newly introduced example can pass the tests?

[1] https://github.com/yunfengzhou-hub/feathub/actions/runs/4476632078

Reply:
Since we have no documents to address that only one processor is allowed to be installed in the same environment, I think we can keep it installing both spark and flink, If any conflict really happens we can quickly find and fix it.